website/src/pages/choropleth/index.js

```diff
@@ -29,6 +29,7 @@ const initialProperties = {
     },
 
     colors: 'nivo',
+    domain: [0,1000000],
```

Comment:
It should be added to the canvas implementation too

Reply:
Done!